2nd-gen/alerts-to-discord/functions/index.js

```diff
@@ -91,4 +95,45 @@ id: \`${id}\`
     );
   }
 });
+
+/**
+ * function triggered by AppDistribution that publishes a message
+ * to Discord whenever a new iOS tester device is registered.
+ */
+// [START v2AppDistributionAlertTrigger]
+exports.postnewduuidtodiscord = onNewTesterIosDevicePublished(async (event) => {
+// [END v2AppDistributionAlertTrigger]
+  // [START v2AppDistributionEventPayload]
+  // construct a helpful message to send to Discord
+  const {
+    testerDeviceIdentifier,
+    testerDeviceModelName,
+    testerEmail,
+    testerName,
+  } = event.data.payload;
+  const message = `
```

Comment:
Is App Distro alert app specific? It is important for users to know which app the new device is registered for?

Reply:
Yes it is. Good catch! The Crashlytics is also app specific. I've added the `appId` to both messages.